src/platform/input/game-pads.js

```diff
@@ -74,56 +115,601 @@ const MAPS = {
             'PAD_R_STICK_X',
             'PAD_R_STICK_Y'
         ]
+    },
+
+    DEFAULT_XR: {
+        buttons: [
+            // Back buttons
+            'XRPAD_TRIGGER',
+            'XRPAD_SQUEEZE',
+
+            // Axes buttons
+            'XRPAD_TOUCHPAD_BUTTON',
+            'XRPAD_STICK_BUTTON',
+
+            // Face buttons
+            'XRPAD_A',
+            'XRPAD_B'
+        ],
+
+        axes: [
+            // Analogue Sticks
+            'XRPAD_TOUCHPAD_X',
+            'XRPAD_TOUCHPAD_Y',
+            'XRPAD_STICK_X',
+            'XRPAD_STICK_Y'
+        ]
     }
 };
 
 const PRODUCT_CODES = {
     'Product: 0268': 'PS3'
 };
 
+let deadZone = 0.25;
+const getGamepads = navigator ? navigator.getGamepads || navigator.webkitGetGamepads : undefined;
```

Comment:
How does this work for you? `getGamepads` is now unbound and throws `Uncaught TypeError: Illegal invocation`:

![image](https://user-images.githubusercontent.com/5236548/217805859-418bfc44-5300-4a84-9c64-c75388288706.png)

I would still rather like it the way I did it in my PR, but you can also just bind it to navigator or rework it in another way.

Reply:
That's a very good question, in fact it was not, I forgot to remove this change when updating the file... Thanks for checking!